internal/receiver/signalfxgatewayprometheusremotewritereceiver/mock_reporter_test.go

```diff
@@ -73,7 +73,7 @@ func (m *mockReporter) OnError(_ context.Context, errorLocation string, err erro
 }
 
 func (m *mockReporter) OnMetricsProcessed(_ context.Context, numReceivedMessages int, _ error) {
-	m.TotalSuccessMetrics.Add(int32(numReceivedMessages))
+	m.TotalSuccessMetrics.Add(int32(numReceivedMessages)) //nolint:gosec
```

Comment:
This is in test code.

@hughesjj the field is not actually used anywhere I can remove it, in a separate PR, unless you have some reason to keep it.